### DISCUSSION TITLE
doris设置字段默认当前时间，手动加入数据没生效

### DISCUSSION THREAD

**Discussion by echobigdata** [apache/doris#25241] [A - General / Q&A]
doris2.0.2
modify_time datetime NULL DEFAULT CURRENT_TIMESTAMP
手动在表中写入数据，modify_time字段没有自动插入当前时间

**Comment by liugddx** [comment]
what version are you using?

**Reply by echobigdata** [reply]
doris-2.0.2

**Reply by yagagagaga** [reply]
What's your insert statement?
